esx_service/cli/vmdkops_admin.py

```diff
@@ -955,9 +958,13 @@ def tenant_access_set(args):
     if args.volume_totalsize:
         volume_totalsize_in_MB = convert.convert_to_MB(args.volume_totalsize)
 
+    if not validate_create_value(str(args.allow_create)):
```

Comment:
Correct. Since Python itself is not type safe, it's possible that non-boolean or other invalid values are passed in.

The AdminCLI commands (including both "tenant access add" and "tenant access set") accept any user input, which may contain invalid values because of typo. So we need to check both methods in auth_api.